docs/resources/Ledger.md

```diff
@@ -63,12 +65,12 @@ go version go1.14.5 darwin/amd64
 
 ### Add your Ledger key
 
-1. Connect and unlock your Ledger device.
-2. Open the Persistence app on your Ledger.
+1. Connect and unlock your **Ledger** device.
+2. Open the Persistence **Wallet** app on your Ledger.
```

Comment:
its not a wallet app. its just a ledger app.

Reply:
Let's change as discussed: Open the Persistence application on your Ledger device.